examples/features/vl3-dns/README.md

```diff
@@ -41,14 +41,17 @@ nscs=$(kubectl  get pods -l app=alpine -o go-template --template="{{range .items
 [[ ! -z $nscs ]]
 ```
 
-Ping each client by each client via DNS:
+Ping each client by each client via DNS. Also check PTR requests:
```

Comment:
Can we move PTR check to the separate section?

Reply:
Fixed